packages/kit/CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # @sveltejs/kit
 
+## 1.2.0
+
+### Minor Changes
+
+- feat: add `text(...)` helper for generating text responses ([#8371](https://github.com/sveltejs/kit/pull/8371))
+
+- feat: enable access to public env within app.html ([#8449](https://github.com/sveltejs/kit/pull/8449))
+
+### Patch Changes
+
```

Comment:
#8371 should probably have been a minor rather than a patch. we can just edit the changelog

```suggestion
```